src/app/screens/connect/btcSelectAddressScreen/index.tsx

```diff
@@ -0,0 +1,292 @@
+
+const TitleContainer = styled.div({
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'center',
+  justifyContent: 'center',
+  overflow: 'hidden',
+  marginLeft: 30,
+  marginRight: 30,
+  marginTop: 48,
+});
+
+const OuterContainer = styled(animated.div)({
+  display: 'flex',
+  flexDirection: 'column',
+  marginLeft: 16,
+  marginRight: 16,
+});
+
+const AddressBoxContainer = styled.div((props) => ({
+  display: 'flex',
+  flexDirection: 'column',
+  width: '100%',
+  marginTop: props.theme.spacing(12),
+}));
+
+const AddressBox = styled.div((props) => ({
+  width: 328,
+  height: 66,
+  padding: props.theme.spacing(10),
+  display: 'flex',
+  flexDirection: 'row',
+  alignItems: 'center',
+  justifyContent: 'space-between',
+  backgroundColor: props.theme.colors.elevation6_800,
+  marginBottom: 1,
+  ':first-of-type': {
+    borderTopLeftRadius: props.theme.radius(2),
+    borderTopRightRadius: props.theme.radius(2),
+  },
+  ':last-of-type': {
+    borderBottomLeftRadius: props.theme.radius(2),
+    borderBottomRightRadius: props.theme.radius(2),
+  },
+}));
+
+const AddressContainer = styled.div({
+  display: 'flex',
+  alignItems: 'center',
+});
+
+const TopImage = styled.img({
+  maxHeight: 32,
+  maxWidth: 32,
+});
+
+const Title = styled.h1((props) => ({
+  ...props.theme.typography.body_bold_l,
+  color: props.theme.colors.white_0,
+  marginTop: 12,
+}));
+
+const DapURL = styled.h2((props) => ({
+  ...props.theme.typography.body_medium_m,
+  color: props.theme.colors.white_400,
+  marginTop: props.theme.spacing(2),
+  textAlign: 'center',
+}));
+
+const AddressImage = styled.img({
+  width: 24,
+  height: 24,
+  marginRight: 8,
+});
+
+const AddressTextTitle = styled.h2((props) => ({
+  ...props.theme.typography.body_medium_m,
+  color: props.theme.colors.white_200,
+  textAlign: 'center',
+}));
+
+const TruncatedAddress = styled.h3((props) => ({
+  ...props.theme.typography.body_medium_m,
+  color: props.theme.colors.white_0,
+  textAlign: 'right',
+}));
+
+const BnsName = styled.h3((props) => ({
+  ...props.theme.typography.body_medium_m,
+  color: props.theme.colors.white_0,
+  textAlign: 'right',
+}));
+
+const RequestMessage = styled.p((props) => ({
+  ...props.theme.typography.body_medium_m,
+  color: props.theme.colors.white_200,
+  textAlign: 'left',
+  wordWrap: 'break-word',
+  marginTop: props.theme.spacing(12),
+  marginBottom: props.theme.spacing(12),
+}));
+
+const PermissionsTitle = styled.h3((props) => ({
+  ...props.theme.typography.body_medium_m,
+  color: props.theme.colors.white_200,
+  textAlign: 'left',
+  marginTop: 24,
+}));
+
+const Permission = styled.div((props) => ({
+  ...props.theme.typography.body_medium_m,
+  color: props.theme.colors.white_0,
+  marginTop: 12,
+  display: 'flex',
+  alignContent: 'center',
+}));
+
+const PermissionIcon = styled.div({
+  marginRight: 4,
+});
+
+const ActionsContainer = styled(StickyHorizontalSplitButtonContainer)({
+  marginTop: 'auto',
+});
```

Comment:
margin bottom looks too low compared to figma

Reply:
nit: I meant the padding-bottom was too low. according to figma, it should be

padding-bottom: 44px;